Tasks/replace.js

```diff
@@ -1,18 +1,20 @@
 // Replace substring with newstr
+'use strict';
+const replace = (string, subString, newString) => {
+  if (subString  ===  '') return string;
```

Comment:
2 spaces just for indentation at the line begin
```suggestion
  if (subString === '') return string;
```